src/test/extensions/extensionsApi.spec.ts

```diff
@@ -192,7 +192,7 @@ describe("extensions", () => {
 
       await extensionsApi.configureInstance(PROJECT_ID, INSTANCE_ID, { MY_PARAM: "value" });
       expect(nock.isDone()).to.be.true;
-    });
+    }).timeout(2000);
```

Comment:
I increased these to 2000 from the default 1000 to fix flakiness that I've noticed.